e2e/atlas/serverless_clusters_test.go

```diff
@@ -0,0 +1,129 @@
+// Copyright 2021 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// +build e2e atlas,serverless
+
+package atlas_test
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"os/exec"
+	"testing"
+
+	"github.com/mongodb/mongocli/e2e"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"go.mongodb.org/atlas/mongodbatlas"
+)
+
+func TestServerlessClusters(t *testing.T) {
+	cliPath, err := e2e.Bin()
+	req := require.New(t)
+	req.NoError(err)
+
+	clusterName, err := RandClusterName()
+	req.NoError(err)
+
+	t.Run("Create", func(t *testing.T) {
+		cmd := exec.Command(cliPath,
+			atlasEntity,
+			serverlessClustersEntity,
+			"create",
+			clusterName,
+			"--region=US_EAST_1",
+			"--provider=AWS",
+			"-o=json")
+		cmd.Env = os.Environ()
+		resp, err := cmd.CombinedOutput()
+		req.NoError(err)
```

Comment:
@fmenezes while you fix the tests on your PR can you fix this line, we are losing the command output which is extremely helpful, change this to `req.NoError(err, string(resp))` to get the command output, same for other commands that disregard it